inference-engine/samples/benchmark_app/inputs_filling.cpp

```diff
@@ -270,6 +279,11 @@ void fillBlobs(const std::vector<std::string>& inputFiles,
                 if (!imageFiles.empty()) {
                     // Fill with Images
                     fillBlobImage(inputBlob, imageFiles, batchSize, app_info, requestId, imageInputId++, imageInputCount);
+#ifdef USE_REMOTE_MEM
+                    if (preallocImage) {
+                        remoteContextHelper.preallocRemoteMem(requests.at(requestId), item.first, inputBlob);
+                    }
+#endif
```

Comment:
What is about other cases - filing inputs from binary files, for example?

Reply:
The 'if case' means it will allocate remote memory and copy the image data to the allocated remote memory, then each infer request will directly use the data without copy. If the 'if case' does not meet each infer request will copy the data from ia host to vpu device.
The 'if case' simulates the video workload, decoding in vpu device then inferring in vpu either without video data copying.